M/MPICH/build_tarballs.jl

```diff
@@ -5,9 +5,7 @@ include(joinpath(YGGDRASIL_DIR, "platforms", "mpi.jl"))
 
 name = "MPICH"
 version_str = "4.2.2"
-version = VersionNumber(version_str)
-
-# build trigger: 1
+version = v"4.2.3"              # We updated the Hwloc compat bounds
```

Comment:
Wait for the real 4.2.3?  [rc1 was released one hour ago](https://github.com/pmodels/mpich/releases/tag/v4.2.3rc1)

Reply:
Can do.